fourth-wall.css

```diff
@@ -81,6 +81,9 @@ body {
 #pulls li.thumbsup {
     background: #5c9317;
 }
+#pulls li.changes-requested {
+    background: #BF1E2D !important;
```

Comment:
I’m not seeing it here: 
![image](https://cloud.githubusercontent.com/assets/355079/22201992/fcd76d0a-e15d-11e6-9d93-51fc04add965.png)

Are you loading in some custom CSS in your gist? This specific line could be the problem: https://gist.github.com/robyoung/91398cee2b6f94e98b54#file-custom-styles-css-L35